test/unit/org/apache/cassandra/schema/TableParamsTest.java

```diff
@@ -36,17 +36,17 @@ public class TableParamsTest
     public void serdeLatest()
     {
         DataOutputBuffer output = new DataOutputBuffer();
-        qt().forAll(tableParams()).checkAssert(FailingConsumer.orFail(params -> {
+        qt().forAll(tableParams()).check(params -> {
```

Comment:
accord `qt` will show the seed that failed when generators fail, but QuickTheories doesn't; making it hard to reproduce these type of errors.

I plan to migrate all tests off of QuickTheories *after* we merge to trunk, and these are cep-15-accord only tests....